gms/feature_service.hh

```diff
@@ -66,8 +66,11 @@ class feature_service final : public peering_sharded_service<feature_service> {
     std::unordered_map<sstring, std::reference_wrapper<feature>> _registered_features;
 
     feature_config _config;
+    bool _use_raft_cluster_features;
+
+    future<> enable_features_on_startup(db::system_keyspace&);
 public:
-    explicit feature_service(feature_config cfg);
+    explicit feature_service(feature_config cfg, bool use_raft_cluster_features);
```

Comment:
nit: Usually bool_class<> is used for such cases to prevent code reader from wondering what `feature_service f(..., false);` might mean.

Reply:
Or put it into feature_config.

Reply:
moved it to `feature_config`